Gemfile.lock

```diff
@@ -250,25 +245,25 @@ GEM
     method_source (0.9.2)
     mime-types (3.3)
       mime-types-data (~> 3.2015)
-    mime-types-data (3.2019.0904)
+    mime-types-data (3.2019.1009)
     mimemagic (0.3.3)
     mini_magick (4.9.5)
     mini_mime (1.0.2)
     mini_portile2 (2.4.0)
-    minitest (5.12.2)
+    minitest (5.13.0)
```

Comment:
### [minitest](https://github.com/seattlerb/minitest)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 5.12.2 → 5.13.0

[_~~change-log~~, ~~source-code~~_]





<details>
<summary>Commits</summary>

A change of **12** commits. See the full changes on [the compare page](https://github.com/seattlerb/minitest/compare/v5.12.2...v5.13.0).

These are the first 10 commits:
- (15ed8e4) [- Improved documentation for _/value/expect, especially for …](https://github.com/seattlerb/minitest/commit/15ed8e4ce504c8313058a1d6fc4918299be34328)
- (2716a85) [- Support new Proc#to_s format. (ko1)](https://github.com/seattlerb/minitest/commit/2716a8505e00c2a00a2f7a22bae6eb85e1eebd29)
- (d80a4d4) [+ Added examples to documentation for assert_raises. (lxxxvi…](https://github.com/seattlerb/minitest/commit/d80a4d49e54e599daba23e45f6b6b15ad21f6609)
- (9d8064c) [- Fix autorun bug that affects fork exit status in tests. (d…](https://github.com/seattlerb/minitest/commit/9d8064c8d894510450942109b75801d4321dc596)
- (e761784) [+ Refactored and pulled Assertions#things_to_diff out of #di…](https://github.com/seattlerb/minitest/commit/e761784b231faad82d23dc88dc55ff26acc76a06)
- (158bd6c) [+ Finally added assert_path_exists and refute_path_exists. (…](https://github.com/seattlerb/minitest/commit/158bd6c36a207d4cab3abd5e5043f9a83416df94)
- (dcac10a) [re-sorted assertions after path additions](https://github.com/seattlerb/minitest/commit/dcac10a1bc8b6a4dac38a38ab80119a742bc11c5)
- (c07a593) [+ Added expectations #path_must_exist and #path_wont_exist. …](https://github.com/seattlerb/minitest/commit/c07a59317238ea6f080914fa23d69da9d2dc4845)
- (fe71663) [Reworked some of metametameta to be more flexible.](https://github.com/seattlerb/minitest/commit/fe716639102cde9daa192f99abca66aa496c0f53)
- (87806c1) [+ Added skip_until(year, month, day, msg) to allow deferring…](https://github.com/seattlerb/minitest/commit/87806c168a9530c7ad110d1d737c0acd1197e161)


</details>